### DISCUSSION TITLE
TinyEngine中如何使用vue-draggable-next

### DISCUSSION THREAD

**Discussion by stwflyfox** [opentiny/tiny-engine#844] [Q&A]
如题， 如何添加到物料中去并使用？

**Comment by chilingling** [comment]
类似正常添加物料的流程即可？

1. 确认配置 npm 信息，版本、导出方式、 js、css
2. 确认属性配置，用于右侧属性面板的配置
3. 配置该组件为容器类型，以及声明拥有的插槽。

按照流程走下来如果有问题，请反馈
